packages/plugins/accounts-manager/src/index.ts

```diff
@@ -1,66 +1,27 @@
 import async from "async";
-import { Embark, Events } /* supplied by @types/embark in packages/embark-typings */ from "embark";
+import { Callback, Embark, Events } /* supplied by @types/embark in packages/embark-typings */ from "embark";
 import { __ } from "embark-i18n";
 import { AccountParser, dappPath } from "embark-utils";
 import { Logger } from 'embark-logger';
 import Web3 from "web3";
-const { blockchain: blockchainConstants } = require("embark-core/constants");
 
 import fundAccount from "./fundAccount";
 
-function arrayEqual(arrayA: string[], arrayB: string[]) {
-  if (arrayA.length !== arrayB.length) {
-    return false;
-  } else {
-    return arrayA.every((address, index) => Web3.utils.toChecksumAddress(address) === Web3.utils.toChecksumAddress(arrayB[index]));
-  }
-}
-
 export default class AccountsManager {
   private readonly logger: Logger;
   private readonly events: Events;
-  private accounts: any[] = [];
-  private nodeAccounts: string[] = [];
   private _web3: Web3 | null = null;
-  private ready = false;
-  private signTransactionQueue: any;
-  private nonceCache: any = {};
+  private _accounts: any[] | null = null;
 
   constructor(private readonly embark: Embark, _options: any) {
     this.logger = embark.logger;
     this.events = embark.events;
 
-    this.embark.registerActionForEvent("blockchain:proxy:request", this.checkBlockchainRequest.bind(this));
-    this.embark.registerActionForEvent("blockchain:proxy:response", this.checkBlockchainResponse.bind(this));
-
+    this.parseAndFundAccounts();
     this.events.on("blockchain:started", () => {
       this._web3 = null;
-      this.parseAndFundAccounts(null);
-    });
-    this.embark.registerActionForEvent("accounts:reseted", async (params, cb) => {
```

Comment:
As described in Discord, this is still needed to reset the accounts during the tests.

Reply:
Bad rebase, my fault. Need to put this back in.